diff --git a/DRreco/src/DRcalib3D.cpp b/DRreco/src/DRcalib3D.cpp
index 4007386..625aa89 100644
--- a/DRreco/src/DRcalib3D.cpp
+++ b/DRreco/src/DRcalib3D.cpp
@@ -93,7 +93,7 @@ StatusCode DRcalib3D::execute() {
     double scale = pSeg->IsCerenkov(cID) ? m_cherenScale.value() : m_scintScale.value();
 
     // create a histogram to do FFT and fill it
-    std::unique_ptr<TH1D> waveHist = std::make_unique<TH1D>("waveHist","waveHist",m_nbins,m_gateStart,m_gateStart+m_gateL);
+    std::unique_ptr<TH1D> waveHist = std::make_unique<TH1D>("waveHist","waveHist",m_nbins,m_gateStart,m_gateStart.operator+(m_gateL));
 
     for (unsigned int bin = 0; bin < waveform.centers_size(); bin++) {
       float timeBin = waveform.getCenters(bin);
