diff --git a/src/cpp/CMakeLists.txt b/src/cpp/CMakeLists.txt
index 1b24b0ff..a81e377f 100644
--- a/src/cpp/CMakeLists.txt
+++ b/src/cpp/CMakeLists.txt
@@ -479,7 +479,7 @@ IF( BUILD_LCIO_EXAMPLES )
 
   # LCIO SHELL needs curses and readline libraries
   FIND_PACKAGE( Curses QUIET )
-  FIND_LIBRARY( READLINE_LIBRARY readline PATHS /usr/local/lib64 /usr/local/lib )
+  FIND_LIBRARY( READLINE_LIBRARY readline)
   IF( CURSES_LIBRARY AND READLINE_LIBRARY )
     ADD_LCIO_EXAMPLE( lsh )
     TARGET_LINK_LIBRARIES( bin_lsh ${CURSES_LIBRARY} ${READLINE_LIBRARY} )

