diff --git a/cmakemodules/MacroPandoraGeneratePackageConfigFiles.cmake b/cmakemodules/MacroPandoraGeneratePackageConfigFiles.cmake
index 21adbfb..5e0354f 100644
--- a/cmakemodules/MacroPandoraGeneratePackageConfigFiles.cmake
+++ b/cmakemodules/MacroPandoraGeneratePackageConfigFiles.cmake
@@ -3,7 +3,7 @@
 # helper macro for generating project configuration file
 MACRO( PANDORA_GENERATE_PACKAGE_CONFIGURATION_FILES )
 
-    FIND_PATH ( PANDORA_CMAKE_MODULES_PATH "MacroCheckPackageVersion.cmake" ${CMAKE_MODULE_PATH} )
+    SET ( PANDORA_CMAKE_MODULES_PATH  ${CMAKE_INSTALL_PREFIX}/cmakemodules/ )
 
     FOREACH( arg ${ARGN} )
         IF( ${arg} MATCHES "Config.cmake" )

