diff --git a/cmake/PandoraSDKConfigVersion.cmake.in b/cmake/PandoraSDKConfigVersion.cmake.in
index 44d646c..b438223 100644
--- a/cmake/PandoraSDKConfigVersion.cmake.in
+++ b/cmake/PandoraSDKConfigVersion.cmake.in
@@ -10,6 +10,6 @@ SET( ${PACKAGE_FIND_NAME}_VERSION_MINOR @PandoraSDK_VERSION_MINOR@ )
 SET( ${PACKAGE_FIND_NAME}_VERSION_PATCH @PandoraSDK_VERSION_PATCH@ )


-INCLUDE( "@PANDORA_CMAKE_MODULES_PATH@/MacroCheckPackageVersion.cmake" )
+INCLUDE( "MacroCheckPackageVersion" )
 CHECK_PACKAGE_VERSION( ${PACKAGE_FIND_NAME} @PandoraSDK_VERSION@ )


