--- a/CMakeLists.txt.orig	2018-01-26 11:39:47.656606839 +0100
+++ b/CMakeLists.txt	2018-01-26 11:40:06.287769448 +0100
@@ -18,7 +18,9 @@
 configure_file(cmake/fccrun.in ${CMAKE_BINARY_DIR}/fccrun)
 install(FILES ${CMAKE_BINARY_DIR}/fccrun
         DESTINATION .
-        PERMISSIONS WORLD_EXECUTE WORLD_READ
+        PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ 
+                    GROUP_EXECUTE GROUP_READ
+                    WORLD_EXECUTE WORLD_READ
         RENAME run)
 
 gaudi_project(FCCSW HEAD
