diff --git a/MaMar/Plots/KKMakefile b/MaMar/Plots/KKMakefile
index bd39eba..99d2394 100755
--- a/MaMar/Plots/KKMakefile
+++ b/MaMar/Plots/KKMakefile
@@ -58,7 +58,7 @@ CLD         =  $(CXX)
 .f.o:
 	$(F77) $(FFLAGS) -c $<
 .c.o:
-	gcc -c $<
+	cc -c $<
 #================================================================================
 SRCLIST= "../KK2f ../glibk ../jetset ../bornv "
 #----------------------
@@ -112,7 +112,7 @@ Plot1.o: 	HisNorm.h KKsem.h
 MAIN2 = Plot1
 OBJECTS2 = $(MAIN2).o KKsem.o
 $(MAIN2).exe:                $(OBJECTS2) $(EXT_LIBRARY3)
-	g++ -o $(MAIN2).exe  $(OBJECTS2) $(EXT_LIBRARY3)  $(RLIB) $(FLIB)
+	$(CXX) -o $(MAIN2).exe  $(OBJECTS2) $(EXT_LIBRARY3)  $(RLIB) $(FLIB)
 $(MAIN2):	$(MAIN2).exe
 	./$(MAIN2).exe
 #===================================================================================================
