--- a/CMakeLists.txt.orig	2018-03-02 11:01:22.182461971 +0100
+++ b/CMakeLists.txt	2018-03-02 11:02:45.706440497 +0100
@@ -46,6 +46,12 @@
 #--- enable CPack --------------------------------------------------------------
 include(cmake/tricktrackCPack.cmake)

+
+#--- search for Eigen package --------------------------------------------------
+set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+find_package(Eigen3 REQUIRED)
+include_directories(${EIGEN3_INCLUDE_DIR})
+
 #--- target for Doxygen documentation ------------------------------------------
 if(tricktrack_documentation)
   include(cmake/tricktrackDoxygen.cmake)
