diff --git a/cmake/LArPandoraContentConfig.cmake.in b/cmake/LArPandoraContentConfig.cmake.in
index bd078884..498db896 100644
--- a/cmake/LArPandoraContentConfig.cmake.in
+++ b/cmake/LArPandoraContentConfig.cmake.in
@@ -42,7 +42,7 @@ FIND_PATH( LArPandoraContent_INCLUDE_DIRS
 
 
 # ---------- libraries --------------------------------------------------------
-INCLUDE( "@PANDORA_CMAKE_MODULES_PATH@/MacroCheckPackageLibs.cmake" )
+INCLUDE( "MacroCheckPackageLibs" )
 
 # only standard libraries should be passed as arguments to CHECK_PACKAGE_LIBS
 # additional components are set by cmake in variable PKG_FIND_COMPONENTS

