diff --git a/larpandoracontent/LArHelpers/LArMvaHelper.h b/larpandoracontent/LArHelpers/LArMvaHelper.h
index 52fb3c08..27393ad8 100644
--- a/larpandoracontent/LArHelpers/LArMvaHelper.h
+++ b/larpandoracontent/LArHelpers/LArMvaHelper.h
@@ -327,7 +327,7 @@ inline pandora::StatusCode LArMvaHelper::WriteFeaturesToFile(std::ofstream &, co
 template <typename TLIST>
 pandora::StatusCode LArMvaHelper::WriteFeaturesToFileImpl(std::ofstream &outfile, const std::string &delimiter, TLIST &&featureList)
 {
-    for (const MvaFeature feature : featureList)
+    for (const MvaFeature &feature : featureList)
         outfile << feature.Get() << delimiter;

     return pandora::STATUS_CODE_SUCCESS;
@@ -343,7 +343,7 @@ LArMvaHelper::MvaFeatureVector LArMvaHelper::ConcatenateFeatureLists(TLIST &&fea

     LArMvaHelper::MvaFeatureVector featureVector;

-    for (const MvaFeature feature : featureList)
+    for (const MvaFeature &feature : featureList)
         featureVector.push_back(feature);

     LArMvaHelper::MvaFeatureVector newFeatureVector = ConcatenateFeatureLists(std::forward<TLISTS>(featureLists)...);

